_posts/2016-09-13-whats-new.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "August changelog - what's new in Sandstorm"
+author: Asheesh Laroia
+authorUrl: https://github.com/paulproteus
+---
+
+August's most visible change is that when new users join a Sandstorm server, they can skip the
```

Comment:
This sentence is confusing to me. I feel like I only know what you mean because I'm familiar with the change. Why "skip the install process" rather than "apps are installed automatically"?